erpnext/portal/doctype/products_settings/products_settings.py

```diff
@@ -40,4 +40,5 @@ def home_page_is_products(doc, method):
 	home_page_is_products = cint(frappe.db.get_single_value('Products Settings', 'home_page_is_products'))
 	if home_page_is_products:
 		doc.home_page = 'products'
-
+	elif doc.home_page == "products":
```

Comment:
I am not sure if this block should be added. What if someone is unaware of Product Settings and is trying to set the homepage as "products" from Website Settings itself it will keep on changing the homepage to home and the user will get confused from where this is coming. Also currently it throws the following error if you try to set the homepage as "products" for this PR

![image](https://user-images.githubusercontent.com/42651287/88165514-6f3eb380-cc33-11ea-9a94-f7956e429cbc.png)

Reply:
So according to this logic, whenever homepage is set to products and then later "Home Page is Products" is unchecked, it should reset back to home. It won't get triggered any other time. I do not see what the problem is with the logic?

For the error, as you see, I haven't changed anything other than adding a condition that checks whether the current homepage is set to products in this pull request.

Reply:
there is a problem with the logic, apologies. ill fix it.